Dockerfile

```diff
@@ -1,6 +1,16 @@
 FROM olantwin/ship-base:20180627-online
 
-ADD . /FairShip
+# Copy FairShip scripts
+COPY . /FairShip
```

Comment:
Why do you replace the `ADD` here? What are the advantages of `COPY`?

Reply:
This is nothing more than a habit, and I found it in tips https://nickjanetakis.com/blog/docker-tip-2-the-difference-between-copy-and-add-in-a-dockerile.

Reply:
This is a good point. This is indeed a good change!